docs/Change_Log.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## Zone-specific Gateway Resume URLs
```

Comment:
"zone" is not defined here, we might want to name this differently, unless we want to include the specifics of availability zones